ethpm/utils/deployments.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, Generator, List, Tuple
+
+from eth_utils import to_bytes, to_canonical_address, to_tuple
+from web3 import Web3
+
+from ethpm.exceptions import BytecodeLinkingError, ValidationError
+
+
+def get_linked_deployments(deployments: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Returns all deployments found in a chain URI's deployment data that contain link dependencies.
+    """
+    linked_deployments = {
+        dep: data
+        for dep, data in deployments.items()
+        if "runtime_bytecode" in data
+        and "link_dependencies" in data["runtime_bytecode"]
```

Comment:
just noting that you can also do this with `cytoolz.get_in(('runtime_bytecode', 'link_dependencies'], data)` but I'm not sure if it's actually a better approach.